examples/customization/custom-vts-metrics/nginx/README.md

```diff
@@ -68,7 +50,7 @@ default-http-backend-2657704409-qgwdd      1/1       Running   0          2m
 nginx-ingress-controller-873061567-4n3k2   1/1       Running   0          42s
 ```
 
-## Test
+## Result
 
 Check the contents of the configmap is present in the nginx.conf file using:
```

Comment:
Probably wrong description, since the below command will pull the metrics (not the contents of nginx.conf)